src/services/measures/attitude/client/attitude.js

```diff
@@ -39,24 +99,28 @@ Gauges.Widgets.attitude = {
 					}
 				});
 			})
-			.add( this, this.publishReadyEvent )
+			.add( this, function(sequence) {
+				this.publishReadyEvent(sequence);
```

Comment:
Actually, LOL, today I'm verbose (what a news...)
I take back my last comment and I'd like to get back to my first comment :p 
I'm not entirely sure if the call to update after publish could theoretically end up in a race condition (since with publish we start listening for socket.io messages, and the message handler calls update too) but it would probably make me happier having the publish as the very last event, unless we see an issue in calling the zeroing update before publish..